sktime/clustering/kmeans.py

```diff
@@ -401,15 +401,44 @@ class MiniBatchKmeansClustering(KmeansClustering):
     KmeansClustering : k-means clustering without mini-batching
     """
 
-    def __init__(self, n_clusters, max_iter=5, metric=None, tolerance=1e-5, init_strategy='kmeans++',
+    def __init__(self, n_clusters, batch_size=100, max_iter=5, metric=None, tolerance=1e-5, init_strategy='kmeans++',
                  n_jobs=None, initial_centers=None):
         """
-        Constructs a Minibatch k-means estimator. For a detailed argument description, see :class:`KmeansClustering`.
+        Constructs a Minibatch k-means estimator. For a more detailed argument description,
+        see :class:`KmeansClustering`.
+
+        The only additional parameter is batch_size, which determines the maximum sample size if calling
+        :meth:`fit()`.
```

Comment:
is there any need to format the docstrings in a specific way here or is this ad hoc way fine

Reply:
I mean it is either that or listing all the parameters - again. Thought it would be good to avoid duplication, also in docs.

Reply:
yeah i think this is the right choice